modules/Microsoft.SignalRService/signalR/deploy.bicep

```diff
@@ -0,0 +1,215 @@
+@description('Optional. The location for the resource.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the SignalR Service resource.')
+param name string
+
+@description('Optional. The kind of the service.')
+@allowed([
+  'SignalR'
+  'RawWebSockets'
+])
+param kind string = 'SignalR'
+
+@description('Optional. The SKU of the service.')
+@allowed([
+  'Free_F1'
+  'Standard_S1'
+  'Standard_S2'
+  'Standard_S3'
+  'Premium_P1'
+  'Premium_P2'
+  'Premium_P3'
+])
+param sku string = 'Standard_S1'
+
+@description('Optional. The unit count of the resource.')
+param capacity int = 1
+
+@description('Optional. The tags of the resource.')
+param tags object = {}
+
+@description('Optional. The allowed origin settings of the resource.')
+param allowedOrigins array = [
+  '*'
+]
+
+@description('Optional. The disable Azure AD auth settings of the resource.')
+param disableAadAuth bool = false
+
+@description('Optional. The disable local auth settings of the resource.')
+param disableLocalAuth bool = false
```

Comment:
We may want to set this to `true`. Disabling local auth is a best pratice for most services - i.e., avoid anonymous access.

Reply:
Good point we do that for automation accounts